thesis-de/content-de.tex

```diff
@@ -2619,14 +2620,14 @@ \section{Realisierung Grafana}
 Die Automatisierung erfolgt mit einem Puppet Modul. Das Puppet Modul hat
```

Comment:
Nenn das mal "Puppet Profil", nicht "Puppet Modul". wir rufen in unserem profil ja nur andere module auf und definieren nichts eigenens. ich erleuter den unterscheid gleich näher im Puppet abschnitt.